docs/TuxReact.md

```diff
@@ -0,0 +1,457 @@
+
+```javascript
+    var React = require('tux/React');
+    var Todos = require('./Todos');
+
+    var TodoOwner = React.createOwnerClass({
+      getInitialState: function () {
+        return {
+          todos: []
+        };
+      },
+
+      addTodo: function (todo) {
+        this.setState({
+          todos: this.state.todos.concat(todo)
+        });
+      },
+
+      registerOwnerProps: function () {
+        return {
+          this.addTodo.bind(this)
+        };
+      },
+
+      render: function () {
+        return (
+          <div>
+            <Todos todos={this.state.todos} />
+          </div>
+        );
+      }
+    });
+```
+
+This class has the power to automatically share its static properties and methods with its children, which means no more cascading down methods through layers of components. This class also has methods for easily and semantically connecting with `TuxStores`. You can read more about it [here](tuxedojs.org/docs/TuxReact/TuxOwnerClass).
+
+***
+
+#### <a id="TuxedoJS-OwneeClass"></a>5.2) TuxedoJS OwneeClass [#](#TuxedoJS-OwneeClass)
+`Tux` provides an `OwneeClass` that is able to take advantage of its parent's `OwnerProps` to access these shared methods. All `Tux` classes are built off of the `OwneeClass`, which means that all `Tux` classes have access to these tools (including `OwnerClass` components):
+
+```javascript
+    var React = require('tux/React');
+    var Todo = require('./Todo');
+
+    var Todos = React.createOwneeClass({
+      propTypes: {
+        todos: React.PropTypes.array.isRequired
+      },
+
+      //Tux components can specify needed ownerProps just as they can with standard React props
+      nearestOwnerPropTypes: {
+        addTodo: React.PropTypes.func.isRequired
+      },
+
+      handleSubmit: function (e) {
+        e.preventDefault();
+        var todoText = this.refs.todoText.getDOMNode();
+
+        this.nearestOwnerProps.addTodo({
+          text: todoText.value
+        });
+
+        todoText.value = '';
+      },
+
+      render: function () {
+        var TodoArray = this.props.todos.map(function (todo) {
+          return <Todo todo={todo} />
+        });
+
+        return (
+          <div>
+            <form onSubmit={this.handleSubmit}>
+              <input ref="todoText" />
+            </form>
+            { TodoArray }
+          </div>
+        );
+      }
+    });
+```
+
+The `OwneeClass` can automatically access the properties and methods exposed to it by its nearest `OwnerClass` through the `nearestOwnerProps` key. You can read more about it [here](tuxedojs.org/docs/TuxReact/TuxOwneeClass).
+
+***
+
+#### <a id="TuxedoJS-MutableClass"></a>5.3) TuxedoJS MutableClass [#](#TuxedoJS-MutableClass)
+`Tux` provides a `MutableClass` designed to take advantage of the performance boosting tools in `React`. Like the other `Tux` classes it to is built off of the `OwneeClass`:
```

Comment:
Recommend changing the last sentence to 'Like the other `Tux` classes it is also built off of the `OwneeClass`'